src/vm.rs

```diff
@@ -1624,9 +1624,14 @@ impl<'a, T: GuestStagePagingMode> FinalizedVm<'a, T> {
         &self,
         msmt_addr: u64,
         msmt_size: usize,
-        index: usize,
+        index_usize: usize,
```

Comment:
Nit picking, but can we keep the argument variable name (`index`) and call the internal one `index_u8`.